content/en/docs/releases/next-release-preview/index.md

```diff
@@ -10,3 +10,7 @@ in the next release of Spinnaker. These notes will be prepended to the release
 changelog.
 
 ## Coming Soon in Release 1.31
+
+### [doNotEval SpEL helper](https://github.com/spinnaker/spinnaker.io/blob/40557c30515d159abb636a965ff3ea5b6eeb7846/content/en/changelogs/1.30.0-changelog.md?plain=1#L26)
```

Comment:
Sorry if I pointed you in the wrong direction before.  Let's link to the actual website, not the source file.
```suggestion
### [doNotEval SpEL helper](https://spinnaker.io/changelogs/1.30.0-changelog/#donoteval-spel-helper)
```